packages/tsconfig/base.json

```diff
@@ -1,21 +1,23 @@
 {
-  "$schema": "https://json.schemastore.org/tsconfig",
-  "display": "Default",
-  "compilerOptions": {
-    "composite": false,
-    "declaration": true,
-    "declarationMap": true,
-    "esModuleInterop": true,
-    "forceConsistentCasingInFileNames": true,
-    "inlineSources": false,
-    "isolatedModules": true,
-    "moduleResolution": "node",
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "preserveWatchOutput": true,
-    "skipLibCheck": true,
-    "strict": true,
-    "strictNullChecks": true
-  },
-  "exclude": ["node_modules"]
-}
+	"$schema": "https://json.schemastore.org/tsconfig",
```

Comment:
What are these changes for?

Reply:
these are prettier changes ig, let me undo it